dashboard/src/modules/components/OverviewComponent/index.jsx

```diff
@@ -52,59 +52,70 @@ const OverviewComponent = () => {
   const isExpandedClass = expanded.size === 0 ? "not-expanded" : "";
   return (
     <div className="overview-container">
-      <Heading title="Overview" />
-      <Accordion isBordered>
-        <Grid hasGutter>
-          <GridItem span={4}>
-            <AccordionItem>
-              <AccordionToggle
-                onClick={() => {
-                  onToggle("expired");
-                }}
-                isExpanded={expanded.has("expired")}
-                id="expired"
-              >
-                Expiring soon
-              </AccordionToggle>
-              <AccordionContent isHidden={!expanded.has("expired")}>
-                {expiringRuns.length > 0 ? (
-                  <ExpiringSoonComponent />
-                ) : (
-                  <NoExpiringRuns />
-                )}
-              </AccordionContent>
-            </AccordionItem>
-          </GridItem>
-          <GridItem span={8} className="new-runs-container ">
-            <AccordionItem>
-              <AccordionToggle
-                onClick={() => {
-                  onToggle("newRuns");
-                }}
-                isExpanded={expanded.has("newRuns")}
-                id="newRuns"
-              >
-                New and Unmanaged
-              </AccordionToggle>
-              <AccordionContent isHidden={!expanded.has("newRuns")}>
-                {newRuns.length > 0 ? (
-                  <>
-                    <NewRunsHeading />
-                    <NewRunsComponent />
-                  </>
-                ) : (
-                  <EmptyTable />
-                )}
-              </AccordionContent>
-            </AccordionItem>
-          </GridItem>
-        </Grid>
-      </Accordion>
-      <Separator />
-      <Card className={`bordered saved-runs-container ${isExpandedClass}`}>
-        <Heading title="Saved Runs" />
-        {savedRuns.length > 0 ? <SavedRunsComponent /> : <EmptyTable />}
-      </Card>
+      {!loadingDone ? (
+        <div className="dashboard-loading-container">
+          <Spinner isSVG />
+          <h2 className="heading-h2">Preparing dashboard</h2>
+
+          <p>If page doesn`t load, try refreshing it or retrying later.</p>
+        </div>
```

Comment:
Why doesn't this use the `LoadingComponent` which you've worked so hard to set up?